[2022-07-22 12:59:52,372] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-14T00:00:00+00:00 [queued]>
[2022-07-22 12:59:52,437] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-14T00:00:00+00:00 [queued]>
[2022-07-22 12:59:52,439] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:52,441] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 12:59:52,446] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:52,526] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-14 00:00:00+00:00
[2022-07-22 12:59:52,544] {standard_task_runner.py:52} INFO - Started process 757 to run task
[2022-07-22 12:59:52,558] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-14T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp1mjqxmku', '--error-file', '/tmp/tmp6y9hx7jl']
[2022-07-22 12:59:52,561] {standard_task_runner.py:80} INFO - Job 30: Subtask t2
[2022-07-22 12:59:52,618] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 12:59:53,712] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-14T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 12:59:56,660] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-14T00:00:00+00:00
[2022-07-22 12:59:56,668] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 12:59:56,968] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220614T000000, start_date=20220722T125952, end_date=20220722T125956
[2022-07-22 12:59:57,084] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 12:59:57,531] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:17:58,236] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-14T00:00:00+00:00 [queued]>
[2022-07-25 09:17:58,264] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-14T00:00:00+00:00 [queued]>
[2022-07-25 09:17:58,265] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:58,266] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:17:58,270] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:58,350] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-14 00:00:00+00:00
[2022-07-25 09:17:58,357] {standard_task_runner.py:52} INFO - Started process 719 to run task
[2022-07-25 09:17:58,372] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-14T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp3jdxk_iu', '--error-file', '/tmp/tmpgccuiq7x']
[2022-07-25 09:17:58,386] {standard_task_runner.py:80} INFO - Job 31: Subtask t2
[2022-07-25 09:17:58,448] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:17:58,643] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-14T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:17:58,777] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-14T00:00:00+00:00
[2022-07-25 09:17:58,778] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:17:58,821] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220614T000000, start_date=20220725T091758, end_date=20220725T091758
[2022-07-25 09:17:58,855] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:17:58,959] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
